boards/nrf9160dk/include/board.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2021 Mesotic SAS
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_nrf9160dk
+ * @{
+ *
+ * @file
+ * @brief       Board configuration for the nrf9160dk board
+ *
+ * @author      Dylan Laduranty <dylan.laduranty@mesotic.com>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define CLOCK_HFCLK         (32U)           /* set to  0: internal RC oscillator
+                                             *        32: 32MHz crystal */
+#define CLOCK_LFCLK         (3)             /* set to  0: internal RC oscillator
+                                             *         3: High Accuracy oscillator */
```

Comment:
is this an external oscillator?

Reply:
AFAICT, it is an internal oscillator.
It is recommended to use this source for the LTE modem.